front/src/applications/stdcmV2/components/StdcmOperationalPoint.tsx

```diff
@@ -141,8 +147,9 @@ const StdcmOperationalPoint = ({
           label={t('trainPath.ci')}
           value={searchTerm}
           onChange={onInputChange}
+          onBlur={handleBlur}
           autoComplete="off"
-          suggestions={operationalPointsSuggestions}
+          suggestions={suggestionsVisible ? operationalPointsSuggestions : []}
```

Comment:
```suggestion
          suggestions={suggestionsVisible && operationalPointsSuggestions || []} />
```
When we have this kind of line, what do you recommend @emersion ?

Reply:
Personally I find the ternary more straightforward. I find it tricky to remember associativity rules for `&&`/`||` each time.